server/validations/users.js

```diff
@@ -0,0 +1,82 @@
+import { check } from 'express-validator/check';
```

Comment:
This is not exactly what I meant.

I meant you should use some sort of schema validation which can happen in each controller.

While this may work, this seems very complicated to set up. 

consider `yup` or `joi` or writing a validator which can be easily set up/used.

If this works well for you though let me know